EIPS/eip-7480.md

```diff
@@ -83,6 +83,12 @@ Existing instructions for reading other kinds of data implicitly pad with zeroes
 
 It is benefitial to avoid exceptional failures, because compilers can employ optimizations like removing a code that copies data, but never accesses this copy afterwards, but such optimization is possible only if instruction never has other side effects like exceptional abort.
 
+### Lack of `EXTDATACOPY`
+
+`EXTCODECOPY` instruction is deprecated and rejected in EOF contracts and does not copy anything when being called in legacy with an EOF contract as target. A replacement instruction `EXTDATACOPY` has been considered, but decided against in order to reduce the scope of changes.
```

Comment:
Maybe 
```suggestion
`EXTCODECOPY` instruction is deprecated and rejected in EOF contracts and does not copy contract code when being called in legacy with an EOF contract as target. A replacement instruction `EXTDATACOPY` has been considered, but decided against in order to reduce the scope of changes.
```
because it copies fixed `EF00` bytes

Reply:
ah, missed it, good catch